app/cdap/components/Metadata/Home/index.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useState } from 'react';
+import T from 'i18n-react';
+import InputLabel from '@material-ui/core/InputLabel';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import FormControl from '@material-ui/core/FormControl';
+import Search from '@material-ui/icons/Search';
+import FilledInput from '@material-ui/core/FilledInput';
+import styled, { css } from 'styled-components';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+
+const I18N_PREFIX = 'features.MetadataHome';
+
+const Container = styled.div`
+  margin: 0 auto;
+  padding: 0 15px;
+  width: 66.6%;
+`;
+
+const StyledFormControl = styled(FormControl)`
+  margin-top: 150px;
+`;
+
+const SearchTipsTitle = styled.p`
+  color: #3cc801;
+  font-size: 18px;
+  margin: 45px 0 10px 0;
+  border-bottom: 1px solid #999;
+`;
+
+const SearchTipsSubTitle = styled.p`
+  font-size: 15px;
+`;
+
+const SearchTipsList = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+`;
+
+const SearchTips = styled.li`
+  margin: 20px 0;
+`;
+
+const Highlight = styled.span`
+  color: #3cc801;
+`;
+
+const MetadataHome: React.FC = () => {
+  const searchQueryRegex = new RegExp('^(?![*]).*$');
+  const [error, setError] = useState(false);
+
+  const handleSearch = (event) => {
+    const searchQuery = event.target.value;
+    const isValid = searchQueryRegex.test(searchQuery);
+    setError(!isValid);
+    if (event.keyCode === 13 && searchQuery.trim() !== '' && isValid) {
```

Comment:
I think we can do `event.key === 'Enter'` or `event.code === 'Enter'` - i think its much easier/clearer than keycode.
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Reply:
Done